pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1548,19 +1548,29 @@ func useCSIPlugin(vpm *volume.VolumePluginMgr, spec *volume.Spec) bool {
 }
 
 func translateSpec(spec *volume.Spec) (*volume.Spec, error) {
+	if spec.PersistentVolume == nil && spec.Volume == nil {
```

Comment:
nit: I would prefer this error case as the `else` block in the below if/else statements as it is kind of the 3rd condition there